config.m4

```diff
@@ -85,11 +85,6 @@ AC_DEFUN([PHP_BSON_CLOCK],
 
 MONGOC_SYMBOL_SUFFIX="priv"
 
-AC_MSG_CHECKING(PHP version)
-PHP_FOUND_VERSION=`${PHP_CONFIG} --version`
-PHP_FOUND_VERNUM=`echo "${PHP_FOUND_VERSION}" | $AWK 'BEGIN { FS = "."; } { printf "%d", ([$]1 * 100 + [$]2) * 100 + [$]3;}'`
-AC_MSG_RESULT($PHP_FOUND_VERNUM)
-
```

Comment:
I think we should keep this check, as it shows up in the configure output as a good hint of what's going on. Right now we might not use it, but we may be in the future again.